dns-server/src/storage.rs

```diff
@@ -1186,14 +1235,15 @@ mod test {
 
         // Begin an update.
         let before = chrono::Utc::now();
-        let update1 = tc.store.begin_update("my req id", 3).await.unwrap();
+        let update1 =
+            tc.store.begin_update("my req id", 3.into()).await.unwrap();
         let after = chrono::Utc::now();
 
         // Concurrently attempt another update.
         let dummy_record = DnsRecord::Aaaa(Ipv6Addr::LOCALHOST);
         let update2 = DnsConfigParams {
             time_created: chrono::Utc::now(),
-            generation: 1,
+            generation: 1.into(),
```

Comment:
This one might have slipped by?

```suggestion
            generation: Generation::from_u32(1),
```

Reply:
Fixed, thanks.